modules/apps/push-notifications/push-notifications-test/src/testIntegration/resources/com/liferay/push/notifications/sender/test/dependencies/service-account-key.json

```diff
@@ -0,0 +1,12 @@
+{
+	"auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+	"auth_uri": "https://accounts.google.com/o/oauth2/auth",
+	"client_email": "zzzzzzzzzz-zzz99999@zzzzzzzzzzzz-zzzzzzzzzzz.iam.gserviceaccount.com",
+	"client_id": "999999999999999999999",
+	"client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/zzzzzzzzzz-zzz99999%40zzzzzzzzzzzz-zzzzzzzzzzz.iam.gserviceaccount.com",
+	"private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDV4qlI1H79f8Xk\n6Ya7xWjRtIPK3hWCXv8IfXnApmPU/5gNdh3AczplRlxTcMMrpq3oDlDubKtC0JhH\n9pI0qJti67p3GmrtYd1t9MHjssY0z1rqwJEaPFZXUOqfLOB0eGbgdA8MRXq729MT\noXCyiOZEE/AeV2fgtH1O1cIhUKIop+pgpyzU4GuBSZvUdOtd21quGO/b/h/gUdQ0\nDAQBdSPpVcEg0FZB3J9nW0TRPCpLxXije5sOEbDPUbHAxNs2ow5qnXVJKsehVag9\nKIydegDRTHbGY/KImHOlFEYWsTzxD0x4BZ10jPB+jTJF4T+lz6h1v/gQwviyWQ4k\nT1wfOp//AgMBAAECggEAe+e5BfYEMzroG7qlUPYFySZ4CvgpIcOCnxqBg/dgci43\nmeTJS4n1VGI0biovBoSizoIspFLTy/snnNjcziC3E9FXzlPBig9dxas6/3u7kzBk\njQZNiRNXOXIo2VmYFiECCBlz9jnx5g5s8zxdGzBvpRSf0BP2m5gz143Bzqb1qfK9\nkZNpsdTJlJY+hxyd0kRrk7BAX6/Eep8/Py44BCxqK0u+6NUEj2Gq8/OALi9rTr4w\nLArthPsZwEJ+vcivpepgGN3h4m480Skzz8gL/mhL6bBvLU/l/lrWE3AwBW5rqWUV\nXwiSEWePty2ukUzUgJKH99NTflCw4DAmWqn85/i5MQKBgQDtZgqqog12DDcuRYwo\nWTtpBXipZmUHPAVRSe0hQznBGYU8lRr7l6dcbyLSsjI1cvOlUDIW81S54KxvLdmA\n5a2pLU8XxVRhDk4XgXg8WgPCaNer66HcDFTav5CRZLaNWZib2dp+GmSUcx2TZb1X\nenDjGWKJvZZEylEyIDCPg0nd1QKBgQDmpPhg4V3rglDwvQy8SGAk2ZEoQ+N7h7lz\nPb/X86xQN4VAnTah9hq0tgQQ9WJU/FukyNJRhR+AAXTdznZ5ws4chBRYMxINoFp0\nqaVPiEAw0/ZYGyHCq3VlsHWyom1cgddIflsf0whuZM9l6ymEsGZw0PQ7s6n2v9Kg\npXYmPFesgwKBgQDarLQGmV9s7bOmm8mPISLoinGrGPbWwyLizgXwLcx1YtyoAXDp\nb7hqHLUm/53gPhLQpB5/pYTV827iuokvgmGFBIdmjg47mMG2RTxL2167JXgElPhz\niahgFzVDuFGvwmggp3vrzMP5OlB0BbNzcQWrKybpWgmhM/u3AI+6omTYQQKBgQDi\n93ykeaJe24KjJKafselK2f8KTKRTL6DDUwSykkwDNF+g0hStFO363xLpTVBSqXPV\nmPm+1u3EnapPi5jzAUyqqx8KHFerqRzaqQbQWRgx9KP619hFJU0FoLxx6X0NB4i/\nAfjSPG2TlQizFtNoi96DLlzpU0fyVZoEFE9MGR192QKBgEeNsTy6R5flikElVVZm\nSzaZkKN4nAluKYvPbFx/K3bWwhOBXT13SIvwVnSSYCThtzcENYeCzPM+9gtCowYq\nJDEJZ02CmnX5Bt3XTzUEsehTdBHGpXk2qC1YBDI5thyL8KDB9dfzqk4/cHH/Nepz\nc/ilqiM92j706xPr+vGAzYyz\n-----END PRIVATE KEY-----\n",
```

Comment:
Please do not use anything resembling a real key, to avoid false positives when security checks the code for keys. If `test` is a valid string, just use it.